src/utils/displayTo.ts

```diff
@@ -1,33 +1,85 @@
+    (acc, next) =>
+      next.breakpoints ? { ...acc, [next.screen]: formatQuery(next.breakpoints.prev, next.breakpoints.next) } : acc,
+    {},
+  );
 }
 
-export function getMediaQueries(breakpoints: Breakpoints) {
-  if (!isBreakpoint(breakpoints, 'medium') || !isBreakpoint(breakpoints, 'large')) {
-    throw new Error('Invaild breakpoints');
+// create max-width media query including screen size
+export function displayUpTo(screen: ScreenSize) {
+  const screenBreakpoints = getScreenSizeBreakpoints(screen, breakpoints);
+  if (!screenBreakpoints) {
+    throw new Error('Invalid screen size');
   }
+  const query = formatQuery(undefined, screenBreakpoints.next);
+  return formatMedia(query);
+}
 
-  return {
-    small: `(max-width: ${breakpoints.medium - 1}px)`,
-    smallAndMedium: `(max-width: ${breakpoints.large - 1}px)`,
-    medium: `(min-width: ${breakpoints.medium}px) and (max-width: ${breakpoints.large - 1}px)`,
-    mediumAndLarge: `(min-width: ${breakpoints.medium}px)`,
-    large: `(min-width: ${breakpoints.large}px)`,
-  };
+// create min-width media query including screen size
+export function displayFrom(screen: ScreenSize) {
+  const screenBreakpoints = getScreenSizeBreakpoints(screen, breakpoints);
+  if (!screenBreakpoints) {
+    throw new Error('Invalid screen size');
+  }
+  const query = formatQuery(screenBreakpoints.prev, undefined);
+  return formatMedia(query);
 }
 
 export function displayTo(screen: ScreenSize | string): ThemedCssFunction<any> {
-  const query = getMediaQueries(breakpoints)[screen] || screen;
-  const result: ThemedCssFunction<any> = (strings: any, ...interpolations: Array<any>) =>
+  const screenBreakpoints = getScreenSizeBreakpoints(screen as ScreenSize, breakpoints);
+  const query = screenBreakpoints ? formatQuery(screenBreakpoints.prev, screenBreakpoints.next) : screen;
+  return formatMedia(query);
+}
+
+function formatMedia(query: string): ThemedCssFunction<any> {
+  return (strings: any, ...interpolations: Array<any>) =>
     css`
       @media ${query} {
         ${css(strings, ...interpolations)};
       }
     `;
-  return result;
+}
+
+function getScreenSizeBreakpoints(
+  screen: ScreenSize,
+  breakpoints: Breakpoints,
+): { next?: number; prev?: number } | undefined {
+  switch (screen) {
+    case 'small':
+      return { next: breakpoints.medium };
+    case 'smallAndMedium':
+      return { next: breakpoints.large };
+    case 'medium':
+      return { prev: breakpoints.medium, next: breakpoints.large };
+    case 'mediumAndLarge':
+      return { prev: breakpoints.medium };
+    case 'large':
+      return { prev: breakpoints.large, next: breakpoints.xLarge };
+    case 'xLarge':
+      return { prev: breakpoints.xLarge, next: breakpoints.max };
+    case 'max':
+      return { prev: breakpoints.max };
+    default:
+      return undefined;
+  }
+}
+
+export function formatQuery(previousBreakpoint?: number, nextBreakpoint?: number) {
```

Comment:
I suggest changing arguments to object type `{ prev?: number, next?: number }`

Reply:
In the current case, I prefer to not use an object as an argument.

Reply:
In that case, I should always remeber what the first parameter means when calling `formatQuery(undefined, 980)`.

IMO: `formatQuery({ next: 980 }}` loosk better.

Reply:
Got your point, you`re right. I will update this